pdf/model/fields.go

```diff
@@ -661,7 +661,7 @@ func (r *PdfReader) newPdfFieldFromIndirectObject(container *core.PdfIndirectObj
 			if !ok {
 				return nil, errors.New("invalid widget annotation")
 			}
-			widget.Parent = field.GetContainingPdfObject()
```

Comment:
Might be safer to set the `Parent` field as well as the `parent` field in case it's used in other packages as the `Parent` field is public.